modules/internal/ChapelBase.chpl

```diff
@@ -1131,6 +1125,7 @@ module ChapelBase {
     return ir;
   }
   
+  pragma "compiler generated" 
   pragma "donor fn"
   pragma "auto copy fn"
   inline proc chpl__autoCopy(x) return chpl__initCopy(x);
```

Comment:
Ah never mind. There are assumptions about assignment functions when this flag is present. However none of these are assignments.